app/forms/TradingNameForm.scala

```diff
@@ -24,7 +24,7 @@ import utils.StopOnFirstFail.constraint
 object TradingNameForm {
 
   val maxLength: Int = 160
-  val tradingNameRegex: String = """^[0-9a-zA-Z &`\-\\'\\.^]{1,160}$"""
+  val tradingNameRegex = """^[-A-Za-z0-9 '’‘()[\\]{}<>!«»\"?\/\\\\+=%#*&$€£_\\@¥.,:;]{1,105}$"""
```

Comment:
You've changed the max char length to 105 here, but the spec still has it as 160. Did you copy the organisation name regex instead?

Also now that the hyphen is at the start you won't need the two backslashes that used to escape it (before the @ symbol)